holidays/countries/senegal.py

```
@@ -0,0 +1,159 @@
+        self._add_eid_al_adha_day(tr("Aïd al-Adha"))
+
+
+class SN(Senegal):
+    pass
+
+
+class SEN(Senegal):
+    pass
+
+
+class SenegalIslamicHolidays(_CustomIslamicHolidays):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)